heartbeat/monitors/active/tcp/tcp_test.go

```diff
@@ -91,10 +91,6 @@ func setupServer(t *testing.T, serverCreator func(http.Handler) *httptest.Server
 	return server, port
 }
 
-func tcpMonitorChecks(host string, ip string, port uint16, status string) mapval.Validator {
```

Comment:
This helper function had a number of unused arguments and was more confusing than helpful.